app/models/dto/AddressDto.scala

```diff
@@ -58,11 +58,38 @@ object AddressDto {
       "line5" -> optional(text)
         .verifying("error.line5_contains_more_than_35_characters", e => e.fold(true)(_.length <= 35))
         .verifying("error.line5_invalid_characters", e => validateAddressLineCharacters(e)),
-      "postcode" -> text
+      "postcode" -> optional(text)
         .verifying("error.enter_a_valid_uk_postcode", e => e match {
-          case PostcodeRegex(_*) => true
+          case Some(PostcodeRegex(_*)) => true
           case _ => false
         }),
+      "country" -> optional(text),
+      "propertyRefNo" -> optional(nonEmptyText)
+    )(AddressDto.apply)(AddressDto.unapply)
+  )
+
+  val internationalForm = Form(
+    mapping(
+      "line1" -> text
+        .verifying("error.line1_required", _.size > 0)
+        .verifying("error.line1_contains_more_than_35_characters", _.size <= 35)
+        .verifying("error.line1_invalid_characters", e => validateAddressLineCharacters(Some(e))),
+      "line2" -> text
+        .verifying("error.line2_required", _.size > 0)
+        .verifying("error.line2_contains_more_than_35_characters", _.size <= 35)
+        .verifying("error.line2_invalid_characters", e => validateAddressLineCharacters(Some(e))),
+      "line3" -> optional(text)
+        .verifying("error.line3_contains_more_than_35_characters", e => e.fold(true)(_.length <= 35))
+        .verifying("error.line3_invalid_characters", e => validateAddressLineCharacters(e)),
+      "line4" -> optional(text)
+        .verifying("error.line4_contains_more_than_35_characters", e => e.fold(true)(_.length <= 35))
+        .verifying("error.line4_invalid_characters", e => validateAddressLineCharacters(e)),
+      "line5" -> optional(text)
+        .verifying("error.line5_contains_more_than_35_characters", e => e.fold(true)(_.length <= 35))
+        .verifying("error.line5_invalid_characters", e => validateAddressLineCharacters(e)),
+      "postcode" -> optional(text),
+      "country" -> optional(text)
+        .verifying("error.country_required", _.size > 0),
```

Comment:
Should the validation here not be a bit stronger? e.g. comparing against the possible countries we will accept.

Reply:
Yeah echoing this one. When autocomplete on chrome adds the United Kingdom sometimes it lets you press continue and changed the country to 'ARAB EMIRATES (UNITED)'.  The only thing that's come out of testing that I'm concerned about. @swaistle